README.md

```diff
@@ -209,11 +210,21 @@ export default new Vuex.Store({
 })
 ```
 
-#### Namespaced vuex modules
+#### Sending Events  
+
+Events can be sent to the Socket.IO server by calling `this._vm.$socket.emit` from a Vuex mutation or action. Mutation or action names are not subject to the same naming requirements as above. Event data must be sent in string format.
+
+```
+  actions: {
+    emitSocketEvent (data) {
+      this._vm.$socket.emit('eventName', JSON.stringify(data))     
```

Comment:
The API was changed in v4:
```diff
- this._vm.$socket.emit
+ this._vm.$socket.client.emit
```

[Check release docs](https://github.com/probil/vue-socket.io-extended/releases/tag/v4.0.0)